### PR TITLE
AVS 3.0.0 python client updates

### PR DESCRIPTION
**Pull request by rahul-aerospike**
- Updated client version to 3.0.0 everywhere
- Updated AVS version to 0.11.1 everywhere
- Minor update in documentation
- Updated `sentence-transformers` to 3.2.1 ( was getting `cannot import name 'cached_download' from 'huggingface_hub'`)
-

### CODE REVIEW

**Comment by arrowplum** [comment]
Would you merge main? I think there are changes in here from a branch that recently got merged.

**Comment by rahul-aerospike** [comment]
> Would you merge main? I think there are changes in here from a branch that recently got merged.

Resolved the merge conflict. Did you mean resolving merge conflict or something else.

File: quote-semantic-search/quote-search/requirements.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ waitress
 
 # Quote processing
 numpy==1.26.4 # fix issues in dependencies after 2.0.0 release
-sentence-transformers==2.2.2
+sentence-transformers==3.2.1
```
Comment (arrowplum) [inline quote-semantic-search/quote-search/requirements.txt:15]: Do you have any idea why we were using https://github.com/UKPLab/sentence-transformers/releases/tag/v2.2.2 before? This doesn't break anything we are using?

**Review by arrowplum — COMMENTED** [review]
one question regarding dependencies but lgtm. I think we will have to merge this before https://github.com/aerospike/aerospike-vector-search-examples/pull/63

**Comment by arrowplum** [comment]
> > Would you merge main? I think there are changes in here from a branch that recently got merged.
> 
> Resolved the merge conflict. Did you mean resolving merge conflict or something else.

I had seen changes from the other branch. Fixed with merge.

**Review by arrowplum — APPROVED** [review]

File: quote-semantic-search/quote-search/requirements.txt (hunk 1 above)
Comment (rahul-aerospike) [reply quote-semantic-search/quote-search/requirements.txt:15]: Looked into git history.
When this work was done (10/11/2023); the 2.2.2 was the latest version of `sentence-transformers` package. My guess is, this is the reason that specific version(2.2.2) was used then.

link to pypi release history: https://pypi.org/project/sentence-transformers/#history

**Review by dwelch-spike — APPROVED** [review]
LGTM

**Review by hev — APPROVED** [review]
I tested prism only,  and reviewed manually. Approved.

**Pull request merged**